doc/release/upcoming_changes/15685.new_feature.rst

```diff
@@ -0,0 +1,7 @@
+``subok`` option for `np.copy`
+------------------------------
+A new kwarg, ``subok``, was added to `np.copy` to allow users to toggle the 
```

Comment:
```suggestion
A new kwarg, ``subok``, was added to `numpy.copy` to allow users to toggle the 
```

Links don't work if they start with `np` instead of `numpy`.